eng/Tools.props

```diff
@@ -1,12 +1,10 @@
 <Project>
   <PropertyGroup>
     <MSBuildTreatWarningsAsErrors>false</MSBuildTreatWarningsAsErrors>
-    <Language>C#</Language>
-    <!-- Need to keep in sync with CodeAnalysis.targets file. -->
-    <AnalyzerPropsFile>$(ArtifactsToolsetDir)Common\Tools.Analyzers.props</AnalyzerPropsFile>
   </PropertyGroup>
 
-  <Import Project="Analyzers.props" Condition="'$(DotNetBuildFromSource)' != 'true'" />
+  <!-- We need to import this props file which contains PackageReferences to analyzers so that analyzer build assets are imported via the Tools generated props and targets -->
+  <Import Project="Analyzers.props" />
```

Comment:
Why did you remove the condition?

Reply:
Because instead I added the condition into the ItemGroup in `Analyzers.props` where we define the `PackageReferences`... https://github.com/dotnet/corefx/pull/41832/files#diff-69361dd523cb7bdd2e3fbbe90ea4533cR2 -- because we don't want to restore them in analyzer.depproj either, right?

Reply:
nit: please wrap the comment into multiple lines.